### PR TITLE
Fix openlayers build

### PR DESCRIPTION
**Pull request by fredj**
fixes #1184 
fixes #1188

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -52,7 +52,7 @@
     "less-plugin-autoprefix": "1.5.1",
     "less-plugin-clean-css": "1.5.1",
     "nomnom": "1.8.1",
-    "openlayers": "fredj/ol3#afb2084",
+    "openlayers": "fredj/ol3#b49546d",
```
Comment (tsauerwein) [inline package.json]: That branch still includes the change in ol3 that removed the provide for `ol.style.StyleFunction`, right?
See https://github.com/camptocamp/ngeo/pull/1180#issuecomment-218204325

Comment (fredj) [inline package.json]: non, this branch does not include this change: see https://github.com/fredj/ol3/tree/modify_condition_alt

Comment (tsauerwein) [inline package.json]: Yes, but commit https://github.com/fredj/ol3/commit/b49546d is in branch https://github.com/fredj/ol3/tree/modify_condition which includes the change. :)

Comment (fredj) [inline package.json]: oups, forget to push this change ...

**Comment by tsauerwein** [comment]
Thanks! LGTM

**Comment by fredj** [comment]
why travis didn't start ?

**Pull request merged**